default/scripting/species/common/xenophobic.macros

```diff
@@ -6,46 +6,49 @@ XENOPHOBIC_SELF
             activation = Turn high = 0
             effects = GiveEmpireTech name = "CON_CONC_CAMP"
 
-        // Give malus to industry, research and happiness for other species nearby
+        // Stability malus for other species nearby
         EffectsGroup
             scope = Source
             activation = And [
                 Planet
+                VisibleToEmpire empire = Source.Owner
                 Number low = 1 condition = And [
                     Planet
-                    OwnedBy empire = RootCandidate.Owner
-                    Not Or [ 
-                        Species name = RootCandidate.Species
-                        Species name = "SP_EXOBOT"
-                    ]
+                    VisibleToEmpire empire = Source.Owner
```

Comment:
could add `Species` to these subconditions, rather than the `Not Population` ? Might be clearer...

Reply:
I'll do.